plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -135,50 +130,176 @@ function auto_sizes_filter_image_tag( $content, array $parsed_block, WP_Block $b
 /**
  * Modifies the sizes attribute of an image based on layout context.
  *
- * @param int    $id           The image id.
- * @param string $size         The image size data.
- * @param string $align        The image alignment.
- * @param string $resize_width Resize image width.
- * @return string The sizes attribute value.
+ * @since n.e.x.t
+ *
+ * @param int          $id            The image id.
+ * @param string|int[] $size          The image size data.
+ * @param string       $align         The image alignment.
+ * @param string       $resize_width  Resize image width.
+ * @param string       $max_alignment The maximum usable layout alignment.
```

Comment:
The `$size` param seems like it can be a bit more specific:

```suggestion
 * @param int                    $id            The image attachment post ID.
 * @param string|array{int, int} $size          Image size name or array of width and height.
 * @param string                 $align         The image alignment.
 * @param string                 $resize_width  Resize image width.
 * @param string                 $max_alignment The maximum usable layout alignment.
```

Reply:
Sure! I just copied the docblock type from `wp_get_attachment_image_src()`, which is what this is being passed to. I like the description improvement as well.